### DISCUSSION TITLE
Can you access `___graphql` route on build?

### DISCUSSION THREAD

**Discussion by rifqirosyidi** [gatsbyjs/gatsby#30018] [Help] [answered]
Hi,

While I can access the graphql route of `http://localhost:8000/___graphql` on development mode, can you access the same route in production, for example: `https://mywebsite.com/___graphql`, I'm still new to gatsby, can anyone help?

**Comment by LekoArts** [comment, marked as answer]
No, this is only accessible in development :)

**Reply by LekoArts** [reply]
No, you can't
